react/I18n/index.jsx

```diff
@@ -37,13 +37,13 @@ export class I18n extends Component {
 
   // for preact
   // eslint-disable-next-line react/no-deprecated
-  componentWillReceiveProps() {
-    this.UNSAFE_componentWillReceiveProps()
+  componentWillReceiveProps(nextProps) {
+    this.UNSAFE_componentWillReceiveProps(nextProps)
   }
```

Comment:
To avoid forgetting to pass the parameters or having to update it if the signature changes, you can just make a variable assignment.

```
class PreactCompatibleComponent extends React.Component {

  componentWillReceiveProps = this.UNSAFE_componentWillReceiveProps;
  componentWillUpdate = this.UNSAGE_componentWillUpdate;

}
```